src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/InputLocalFinalVariableName.java

```diff
@@ -188,6 +188,26 @@ void veryLong()
           enough talk */
     }
 
+    // check for try-with-resources
+    void method() throws Exception {
+        final String fileName = "Test";
+        try (BufferedReader bR = new BufferedReader(new InputStreamReader(
```

Comment:
@Nimfadora I think you are mis-understanding what I am asking here. We need good cases (no violations) and bad cases (violations) involving try-with-resources. They need to be in their own input file and not modifying an existing one.

Undo the changes to this file. Create a new file like `InputLocalFinalVariableNameTryResources.java`.
Add all your cases to that file and create a test in `LocalFinalVariableNameCheckTest` to run it.

The example above is a good case, no violations.
The 2 examples you had before were, which are now gone, are bad cases, violations.
All 3 should be in this new input file.